src/main/src/main/java/org/geoserver/logging/LoggingUtils.java

```diff
@@ -71,34 +67,42 @@ public static void configureGeoServerLogging(
 
         Properties lprops = new Properties();
         lprops.load(loggingConfigStream);
+
+        // look up log file location
+        if (logFileName == null) {
```

Comment:
This code has moved up... but why bother about logging on this if the file logging has been disabled? 
In some environments (docker+kubernetes I believe? not sure) logging only to stdout is the preferred approach.

Reply:
Was trying for minimal change, and preserve the same logic and messages as previous.

Reply:
Okay that cleaned up nicely.